_posts/_posts/0000-01-02-RickGiannini.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This class is killing me!
![I-AM-NOT-PREPARED-FOR-THIS](https://user-images.githubusercontent.com/79111260/107997671-502db780-6fb1-11eb-92cf-1b9d0f8a02fe.png)

Reply:
ok